runtime/src/main/java/org/corfudb/runtime/exceptions/GarbageCollectorException.java

```diff
@@ -1,15 +0,0 @@
-package org.corfudb.runtime.exceptions;
-
-/**
- *
- * An exception for any exception that is thrown during
- * garbage (on the runtime) collection.
- *
- * Created by maithem on 11/16/18.
```

Comment:
Considering this one is added relatively recently (end of 2018), can you check with @Maithem to confirm it is obsolete?

Reply:
Sure, actually @vjeko removed this exception as part of #1669, but will check with @Maithem too. thanks